wicker/core/storage.py

```diff
@@ -72,6 +82,17 @@ def check_exists_s3(self, input_path: str) -> bool:
         except ClientError:
             return False
 
+    @retry(Exception, tries=get_config().s3_storage_config.retries, backoff=5, delay=4, logger=logging)
+    def download_with_retries(self, bucket: str, key: str, local_path: str, s3_input_path: str):
```

Comment:
Let's drop `s3_input_path` and use `key` instead.

Reply:
Makes sense. Done.